examples/linear_model/plot_lasso_lars_ic.py

```diff
@@ -28,11 +28,6 @@
 #         Guillaume Lemaitre
 # License: BSD 3 clause
 
-# %%
-import sklearn
-
-sklearn.set_config(display="diagram")
-
```

Comment:
Note: it seems that we actually do not display the pipeline diagram in this example but I think this is fine.